src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
Avoid using `System.out.println` in your classes. It's better to return a String from the `draw` method and print it outside the class, for example in the `main` method or another class that handles user interface.